packages/common/src/types.ts

```diff
@@ -31,10 +31,7 @@ export interface Chain {
   consensus?: {
     type: ConsensusType | string
     algorithm: ConsensusAlgorithm | string
-    clique?: {
-      period: number
-      epoch: number
-    }
+    clique?: CliqueConfig
     ethash?: any
     casper?: any
```

Comment:
I would be a fan of also adding `EthashConfig` and `CasperConfig` types here as well, even if they might be empty.

Reply:
Makes sense.  Will update, check all the tests, etc, and then push a new commit.